lua/textcase/plugin/presets.lua

```diff
@@ -100,4 +64,51 @@ M.setup = function(opts)
   })
 end
 
+M.Initialize = function()
+  plugin.register_methods(api.to_upper_case)
+  plugin.register_methods(api.to_lower_case)
+  plugin.register_methods(api.to_snake_case)
+  plugin.register_methods(api.to_dash_case)
+  plugin.register_methods(api.to_title_dash_case)
+  plugin.register_methods(api.to_constant_case)
+  plugin.register_methods(api.to_dot_case)
+  plugin.register_methods(api.to_phrase_case)
+  plugin.register_methods(api.to_camel_case)
+  plugin.register_methods(api.to_pascal_case)
+  plugin.register_methods(api.to_title_case)
+  plugin.register_methods(api.to_path_case)
+  plugin.register_methods(api.to_upper_phrase_case)
+  plugin.register_methods(api.to_lower_phrase_case)
+
+  plugin.register_replace_command("Subs", {
+    api.to_upper_case,
+    api.to_lower_case,
+    api.to_snake_case,
+    api.to_dash_case,
+    api.to_title_dash_case,
+    api.to_constant_case,
+    api.to_dot_case,
+    api.to_phrase_case,
+    api.to_camel_case,
+    api.to_pascal_case,
+    api.to_title_case,
+    api.to_path_case,
+    api.to_upper_phrase_case,
+    api.to_lower_phrase_case,
+  })
+end
+
+M.setup = function(opts)
+  local prefix = opts and opts.prefix or "ga"
+
+  local default_keymappings_enabled = true
+  if opts and opts.default_keymappings_enabled ~= nil then
```

Comment:
There are now enough tests to verify the default key mapping does not break with this change

Thanks for your patience on the LSP and Telescope setups
And for adding n and v keybinding tests